### PR TITLE
qemu.tests.block_stream.with_stress.cancel_async: Update case steps.

### PR DESCRIPTION
**Pull request by CongLi**
Need wait for block stream finished when streaming.

Signed-off-by: Cong Li <coli@redhat.com>

id: 1368013

### CODE REVIEW

File: qemu/tests/cfg/block_stream.cfg [hunk 1]
```diff
@@ -94,4 +94,4 @@
                     cancel_timeout_image1 = 3
                     max_speed_image1 = 10M
                     when_streaming = "pause_job set_speed resume_job cancel"
-                    after_finished = "start reboot verify_alive"
```
Comment (xutian) [inline qemu/tests/cfg/block_stream.cfg]: it looks after_finished should be call after wait_for_finished. if not, I think it a bug for blk_stream module.

Thanks,
Xu

**Comment by suqinhuang** [comment]
Cuold you update the commit log, didn't get the meaning

File: qemu/tests/cfg/block_stream.cfg (hunk 1 above)
Comment (xutian) [inline qemu/tests/cfg/block_stream.cfg]: if so why not move "start wait_for_finished" to when_streaming section.

**Comment by vivianQizhu** [comment]
Looks it can not fix the bz, still hit the login time out error.

File: qemu/tests/cfg/block_stream.cfg (hunk 1 above)
Comment (CongLi) [inline qemu/tests/cfg/block_stream.cfg]: OK, update it to "when_streaming".

**Comment by CongLi** [comment]
@suqinhuang 

commit log has been updated, could you please help check is it ok?

Thanks.

**Comment by CongLi** [comment]
Hi @vivianQizhu 

Test pass on the host you met the login timeout.
 (1/1) smp_8.1024m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.block_stream.with_stress.cancel_async: PASS (1307.94 s)

**Comment by vivianQizhu** [comment]
Retested and it works well.
Acked-by: Qianqian Zhu qizhu@redhat.com

**Comment by CongLi** [comment]
@xutian @suqinhuang 

Could you please help review it again?

Thanks.

**Comment by xutian** [comment]
Looks OK to me

**Pull request merged**